CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Bug Fixes
 
+* **js:** Update JS namespace from Mozilla to Mzp (#253)
```

Comment:
I'd probably call this a bug rather than a feature change, since the current namespace is colliding with bedrock. Bug fixes can't always be non-breaking changes, so I think as long as our next release of Protocol is a major version bump, this should be ok.